controllers/spaceprovisionerconfig/spaceprovisionerconfig_controller.go

```diff
@@ -32,7 +33,7 @@ func (r *Reconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&toolchainv1alpha1.SpaceProvisionerConfig{}, builder.WithPredicates(predicate.GenerationChangedPredicate{})).
 		Watches(
 			&toolchainv1alpha1.ToolchainCluster{},
-			handler.EnqueueRequestsFromMapFunc(MapToolchainClusterToSpaceProvisionerConfigs(r.Client)),
+			mapper.IgnoringTimeChangesInToolchainConditions(handler.EnqueueRequestsFromMapFunc(mapToolchainClusterToSpaceProvisionerConfigs(r.Client))),
```

Comment:
could you please clarify why it's not implemented as a predicate? It looks like a wrapper of the mapper, but a predicate would be a better fit for this.